packages/react-native/ReactCommon/react/renderer/components/view/YogaLayoutableShadowNode.cpp

```diff
@@ -149,12 +139,33 @@ YogaLayoutableShadowNode::YogaLayoutableShadowNode(
         static_cast<const YogaLayoutableShadowNode&>(sourceShadowNode)
             .yogaTreeHasBeenConfigured_;
   }
+                        
+  bool wasDisplayContents = hasDisplayContentsStyle();
 
   if (fragment.props) {
     updateYogaProps();
   }
 
-  if (fragment.children) {
+  bool isDisplayContents = hasDisplayContentsStyle();
+                        
+  if (!getTraits().check(ShadowNodeTraits::Trait::LeafYogaNode) && !isDisplayContents) {
+    for (auto& child : getChildren()) {
+      if (auto layoutableChild = std::dynamic_pointer_cast<const YogaLayoutableShadowNode>(child)) {
+        runForEveryConcreteSubtree(layoutableChild, [&](const YogaLayoutableShadowNode::Shared subtreeRoot) {
```

Comment:
```suggestion
        runForEveryConcreteSubtree(layoutableChild, [&](const YogaLayoutableShadowNode::Shared& subtreeRoot) {
```

Reply:
Done in https://github.com/facebook/react-native/pull/46584/commits/bc1984f5c3ddc2a529c842377a2dbecb7c679dab